src/js/components/Breadcrumbs.js

```diff
@@ -1,16 +1,16 @@
 import React, {PropTypes} from 'react';
 
-import ManualBreadcrumbs from './ManualBreadcrumbs';
+// import ManualBreadcrumbs from './ManualBreadcrumbs';
```

Comment:
I guess you are commenting out in order to keep the logic in case we introduce breadcrumbs in the future, I am OK with this. 👍